test/utils/manifests.go

```diff
@@ -86,12 +86,40 @@ func TestingMachineProviderSpec(awsCredentialsSecretName string, clusterID strin
 		},
 		PublicIP: &publicIP,
 	}
+}
+
+func TestingMachineProviderSpec(awsCredentialsSecretName string, clusterID string) (clusterv1alpha1.ProviderSpec, error) {
+	machineProviderSpec := testingAWSMachineProviderSpec(awsCredentialsSecretName, clusterID)
+	codec, err := providerconfigv1.NewCodec()
+	if err != nil {
+		return clusterv1alpha1.ProviderSpec{}, fmt.Errorf("failed creating codec: %v", err)
+	}
+	config, err := codec.EncodeProviderSpec(machineProviderSpec)
+	if err != nil {
+		return clusterv1alpha1.ProviderSpec{}, fmt.Errorf("EncodeToProviderConfig failed: %v", err)
+	}
+	return *config, nil
+}
 
+func TestingMachineProviderSpecWithEBS(awsCredentialsSecretName string, clusterID string) (clusterv1alpha1.ProviderSpec, error) {
+	volumeSize := int64(16384) // MBs
```

Comment:
as we sure this is MBs? from the aws API sdk
```
	// The size of the volume, in GiB.
	//
	// Constraints: 1-16384 for General Purpose SSD (gp2), 4-16384 for Provisioned
	// IOPS SSD (io1), 500-16384 for Throughput Optimized HDD (st1), 500-16384 for
	// Cold HDD (sc1), and 1-1024 for Magnetic (standard) volumes. If you specify
	// a snapshot, the volume size must be equal to or larger than the snapshot
	// size.
	//
	// Default: If you're creating the volume from a snapshot and don't specify
	// a volume size, the default is the snapshot size.
	VolumeSize *int64 `locationName:"volumeSize" type:"integer"`
```

Reply:
Yep, was getting it from here: https://github.com/openshift/cluster-api-provider-aws/pull/137

Changed that to something more realistic.